notifier.rst

```diff
@@ -143,6 +144,11 @@ RocketChat  ``symfony/rocketchat-notifier``  ``rocketchat://TOKEN@ENDPOINT?chann
     5.1. The Slack DSN changed in Symfony 5.1 to use Slack Incoming
     Webhooks instead of legacy tokens.
 
+.. versionadded:: 5.2
+
+    The LinkedIn integration was introduced in Symfony
+    5.2.
```

Comment:
I think we can merge this on one line :) (we don't have a very strict max line length rule, it should break somewhere around the 72th character)

Reply:
This was done while merging. Thanks!